src/appleseed.shaders/include/alshaders/noise.h

```diff
@@ -0,0 +1,367 @@
+
```

Comment:
Ok. I didn't know if we had more uses of it or not. That's why I put it in its own header,